frontend/app/work_packages/directives/summarized-cost-entries-directive.js

```diff
@@ -28,7 +28,7 @@
 
 angular.module('openproject.workPackages.directives')
 
-.directive('summarizedCostEntries', ['PathHelper', function(PathHelper) {
+.directive('summarizedCostEntries', function(PathHelper, CostTypeService) {
```

Comment:
'CostTypeService' is defined but never used.<br>'PathHelper' is defined but never used.